### ISSUE TITLE
ToggleControl label misalignment when used outside of InspectorControls

### ISSUE THREAD

**Issue by simonhammes** [WordPress/gutenberg#17127]
**Describe the bug**
When a ToggleControl element is used outside of an InspectorControls element, the supplied label is slightly heightwise misaligned. I discovered this bug using Gutenberg 6.3, but it occurs in 5.4 (is that the version shipped in WP 5.2.2?) as well.

**Code**
``` jsx
<ToggleControl
    label="Display authors"
    checked={ props.attributes.isToggled }
    help={ props.attributes.isToggled ? 'Authors will be displayed' : 'Authors will not be displayed' }
    onChange={ () => props.setAttributes( { isToggled: ! props.attributes.isToggled } ) }
/>
<InspectorControls>
    <PanelBody title="Settings">
        <ToggleControl
            label="Display authors"
            checked={ props.attributes.isToggled }
            help={ props.attributes.isToggled ? 'Authors will be displayed' : 'Authors will not be displayed' }
            onChange={ () => props.setAttributes( { isToggled: ! props.attributes.isToggled } ) }
        />
    </PanelBody>
</InspectorControls>
```

**Screenshots**
![Screenshot_2019-08-21 Edit Page ‹ Mein Blog — WordPress](https://user-images.githubusercontent.com/10352679/63448744-e6c1fc80-c43e-11e9-901d-c4eb9e0b5f57.png)

**Desktop:**
 - OS: Windows
 - Browser: Firefox 68 and Chrome 76

**Comment by simonhammes** [comment]
It seems like the cause of this behaviour is related to #17129. 
Again, the inheritance of style properties from the '.editor-styles-wrapper' class causes issues, namely the line-height of the text (in this case 'Display authors') being set to 1.8. If the element wouldn't inherit this property, the line-height would be set to 1.4em which fixes the issue (for me at least).
Maybe someone else can double check  that.

**Comment by glendaviesnz** [comment]
I confirmed that it is the inherited line height from `.editor-styles-wrapper` that causes this misalignment. A suggested fix at #18815
